perfarce.py

```diff
@@ -220,15 +223,28 @@ def __del__(self):
         except Exception:
             pass
 
+def int_to_bytes(x: int) -> bytes:
```

Comment:
inline type informations are python 3 only.  A portable alternative would be using stub (.pyi) type files